src/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ endif()
 
 # Base directories
 set(PROJECT_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/..")
-set(ESODBC_SRC "${CMAKE_CURRENT_SOURCE_DIR}/elasticodbc")
+set(ESODBC_SRC "${CMAKE_CURRENT_SOURCE_DIR}/odfesqlodbc")
 set(ESENLIST_SRC "${CMAKE_CURRENT_SOURCE_DIR}/elasticenlist")
```

Comment:
Rename this project as well (something like `ODFEENLIST_SRC`)

Reply:
Renamed project